uw-frame-components/portal/search/partials/search.html

```diff
@@ -7,7 +7,7 @@
             aria-label="Search bar enter the app you are looking for"
             focus-me="showSearchFocus"
             ng-show="$storage.typeaheadSearch"
-            typeahead="title as match.title for match in filterMatches | limitTo:10"
+            uib-typeahead="title as match.title for match in filterMatches | limitTo:10"
```

Comment:
Is `typeahead` broken if you don't update it?  We're using this in STAR, so to us it would be a major/breaking change if so, or a minor/non-breaking change if not.